docs/topics/serialization.md

```diff
@@ -157,4 +162,15 @@ fun main() {
 }
 ```
 
+## What's next
+
 For more information about serialization in Kotlin, see the [Kotlin Serialization Guide](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serialization-guide.md).
+
+You can further explore various aspects of Kotlin serialization in the following resources:
+
+* [Learn more about Kotlin serialization and its core concepts.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/basic-serialization.md)
```

Comment:
Can you remove the full stops at the end of these sentences since they are all links? This is something we need to add to our style guidelines.